packages/devextreme/js/ui/text_box/ui.text_editor.base.js

```diff
@@ -493,8 +493,8 @@ const TextEditorBase = Editor.inherit({
     },
 
     _setFieldAria(force) {
-        const { 'aria-label': ariaLabel } = this.option('inputAttr');
-
+        const inputAttributeOption = this.option('inputAttr');
+        const ariaLabel = inputAttributeOption && inputAttributeOption['aria-label'];
```

Comment:
Try it pls

```suggestion
        const ariaLabel = inputAttr?.['aria-label'];
```

Reply:
this works tested it as well
[0c3f424](https://github.com/DevExpress/DevExtreme/pull/27328/commits/0c3f424ee03517e7723add3750e3182b0ebe7eff)